response_automater/types/streaming_request.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, Field
+from pydantic.functional_validators import AfterValidator
+from typing_extensions import Annotated
+
+
+# TODO: Do we need to keep this check? If so would it be better to just check string length rather than calling split() ?
+def not_over_one_thousand_words(v: str):
+    assert len(v.split(" ")) <= 1000, "Question exceeded 1000 words"
+    return v
+
+
+# For testing validation behaviour.
+# Does not allow the question to hold the value "dessert"
+def no_dessert_allowed(v: str):
+    assert v != "dessert", f"{v} is not allowed"
+    return v
```

Comment:
This is admittedly very noddy, I just wanted an input I could use to check that the `AfterValidator` methods were working without throwing it a body of over 1,000 words. Happy to remove, or change to a value even obscurer than `dessert` if you like

Reply:
I think we could remove this now if your happy that the validation is working, but keep the question length check in place.

Reply:
Agreed - have removed :)